application/frontend/controllers/MfaController.php

```diff
@@ -215,4 +301,41 @@ public function actionUpdate($mfaId)
             throw $e;
         }
     }
+
+    /**
+     * @param $mfaId
+     * @param $webauthnId
+     * @throws NotFoundHttpException
+     * @throws ServiceException
+     */
+    public function actionUpdateWebauthn($mfaId, $webauthnId)
+    {
+        $label = \Yii::$app->request->getBodyParam('label');
+        if ($label === null) {
+            \Yii::$app->response->statusCode = 204;
```

Comment:
I just noticed that this returns a `204` if no label was provided, but the RAML looks like it says it will return a missing or blank label value. Is that accurate? Which do we want it to do?

Reply:
Good catch. I'm fixing this, although I don't see in the RAML where it says it will return a a missing or blank label value.